src/pages/en/install/auto.md

```diff
@@ -36,62 +37,44 @@ yarn create astro
 pnpm create astro@latest
 ```
 
+The `create-astro` wizard will walk you through setting up a new Astro project, by asking the following questions:
 
-Depending on your package manager, you may be prompted to confirm you want to install `create-astro@latest`. You will then be asked to specify a project folder (e.g. `./my-astro-site`) which will create a new directory.
-
-### Choose a Starter Template
-You will then see a short list of starter templates to choose from:
-- `Just the basics`: A great starter template for anyone wanting to explore Astro.
-- `Blog`, `Documentation`, `Portfolio`: opinionated themes for specific use-cases.
-- `Completely empty`: A template that just includes the bare minimium to get started.
-
-Use the arrow keys (up and down) to navigate to the template you want to install, then press return (enter) to submit.
-
-:::tip[Online previews]
-Want to check out the templates in your browser before choosing? Visit [astro.new](https://astro.new/)
-:::
-
-### Install Dependencies (optional)
-The wizard will offer to run an `install` command for you at this time, which is optional.
+1. Where would you like to create your new Astro project?
+    
+    Astro must be installed in an empty folder. Type in a name for your new project directory.
+    
+    ```bash
+    ./my-astro-site
+    ```
 
-:::caution
-If you do not do so at this time, you will need to [install dependencies](/en/install/auto/#2-install-dependencies) after the wizard has finished, before starting your project.
-:::
-
-### Install any Official Astro Integrations (optional)
-You will be given the option at this time to add any [additional UI frameworks](/en/core-concepts/framework-components/) (React, Svelte, Vue, Solid, Preact, Lit) and then other Astro official integrations (Tailwind, Partytown, Sitemap) by running `astro add --yes`
-
-
-To select which Astro integrations, if any, you would like to include in your project, use the arrow keys (up and down) to navigate and the space bar to toggle between selected states. You can select multiple items at once, or you can continue without selecting any integrations.
-
-
-When you are satisfied with your selection, press return (enter) to submit.
-
-:::note
-These integrations, and any [Astro community integrations](https://astro.build/integrations/), can also be added later by following the instructions in our [integrations guide](/en/guides/integrations-guide/).
-:::
+1. Which template would you like to use? (browse all templates on [astro.new](https://astro.new/)).
```

Comment:
```suggestion
1. Which template would you like to use?
```
We can remove this now that we have links/descriptions below, and we have the tip at the top of the page.